example-app/src/main/AndroidManifest.xml

```diff
@@ -11,8 +11,14 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
+
+        <meta-data
+            android:name="com.hcaptcha.sdk.site-key"
+            android:value="10000000-ffff-ffff-ffff-000000000001" />
```

Comment:
Was this a new requirement to allow the sitekey via manifest meta data? I do not see it being part of Issue #24?

Reply:
This looks like a nice feature indeed, but it can be confusing to users using multiple sitekeys. Can this be moved to a separate PR, or maybe open a ticket for approval first? It would be nice to keep the PR focused on the ticket it is solving.

Reply:
It can be but to be on the same page this feature is completely optional.

SDK will check `meta-data` only in case if user didn't passed it explicitly (if user called `getClient(context)`), so it's not break any existing behavior.

Ticket https://github.com/hCaptcha/hcaptcha-android-sdk/issues/26

Reply:
Understood. It just now gives more options to configure the sitekey. We need to make sure that the siteykey passed via `HCaptchaConfig` has higher precedence over the meta data.

Reply:
For sure, this is how it's done right now but I will add a dedicated test for this also